infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutAssetDao.kt

```diff
@@ -332,6 +277,7 @@ abstract class LayoutAssetDao<T : LayoutAsset<T>>(
           v.id = :id
           and v.change_time <= :moment
           and not v.draft
+          and design_id is null
```

Comment:
fetchOfficialVersionAtMoment()illa ei ole suunnitelmatilaisuuden tukea, mutta parempi silti on, ettei se hämmenny siitä, että taulussa on suunnitelmatilaisia rivejä.

Reply:
Näitä `...AtMoment`-versioita taidetaan käyttää pääasiassa julkaisujen yhteydessä. Tällä hetkellä tämä on ihan OK näin koska meillä ei taida olla näköpiirissä mitään käliä suunnitelmien julkaisujen näyttämiselle, mutta suunnitelmillekin kuitenkin tullaan tekemään julkaisuja. Täten voi olla että nämä pitää extendata tukemaan kunnolla designejakin jossain kohtaa.